src/lib/CrsfProtocol/crsf_protocol.h

```diff
@@ -301,6 +309,51 @@ typedef struct crsf_sensor_baro_vario_s
     int16_t verticalspd;  // Vertical speed in cm/s, BigEndian
 } PACKED crsf_sensor_baro_vario_t;
 
+// CRSF_FRAMETYPE_AIRSPEED
+typedef struct crsf_sensor_airspeed_s
+{
+    uint16_t speed;             // Airspeed in 0.1 * km/h (hectometers/h)
+} PACKED crsf_sensor_airspeed_t;
+
+// CRSF_FRAMETYPE_RPM
+typedef struct crsf_sensor_rpm_s
+{
+    uint8_t rpm_source_id;      // Identifies the source of the RPM data (e.g., 0 = Motor 1, 1 = Motor 2, etc.)
+    int32_t rpm0:24;            // 1 - 19 RPM values with negative ones representing the motor spinning in reverse
+    int32_t rpm1:24;
+    int32_t rpm2:24;
+    int32_t rpm3:24;
+    int32_t rpm4:24;
+    int32_t rpm5:24;
+    int32_t rpm6:24;
+    int32_t rpm7:24;
+    int32_t rpm8:24;
+    int32_t rpm9:24;
+    int32_t rpm10:24;
+    int32_t rpm11:24;
+    int32_t rpm12:24;
+    int32_t rpm13:24;
+    int32_t rpm14:24;
+    int32_t rpm15:24;
+    int32_t rpm16:24;
+    int32_t rpm17:24;
+    int32_t rpm18:24;
+} PACKED crsf_sensor_rpm_t;
+
+// CRSF_FRAMETYPE_TEMP
+typedef struct crsf_sensor_temp_s
+{
+    uint8_t     temp_source_id;       // Identifies the source of the temperature data (e.g., 0 = FC including all ESCs, 1 = Ambient, etc.)
+    int16_t     temperature[20];      // up to 20 temperature values in deci-degree (tenths of a degree) Celsius (e.g., 250 = 25.0°C, -50 = -5.0°C)
+} PACKED crsf_sensor_temp_t;
+
+// CRSF_FRAMETYPE_CELLS
+typedef struct crsf_sensor_cells_s
+{
+    uint8_t Cell_Sensor_source_id;    // Identifies the source of the Main_battery data (e.g., 0 = battery 1, 1 = battery 2, etc.)
+    uint16_t Cell_Sensor_value[29];   // up to 29 cell values in a resolution of a thousandth of a Volt (e.g. 3.850V = 3850)
```

Comment:
These have different CapiTaliZation than all other members, as well as including the word sensor in them for some reason when the others do not. `source_id` and `voltage[29]` (or `cell[29]` if you prefer)

Reply:
Done